examples/nextjs/src/graphql/codegen.yml

```diff
@@ -0,0 +1,11 @@
+schema:
+  - ${GRAFBASE_API_URL}:
+      headers:
+        Authorization: Bearer ${GRAFBASE_API_KEY}
+documents: ./src/graphql/document.graphql
+generates:
+  ./src/graphql/schema/index.ts:
+    plugins:
+      - typescript
+      - typescript-urql
```

Comment:
Not so urgent to prevent merging this but it's highly recommended you use [`typed-document-node`](https://github.com/dotansimha/graphql-typed-document-node) nowadays.

Reply:
Okay, they can co-live, we use some graphql outside of the flow of hooks. Using the TypedDocumentNode will make this painless.

Reply:
The main benefits are that it reduces the overall generated code, and allows for using the Typed DocumentNode w directly with the URQL hooks, or Apollo if someone wanted to switch, very easily. 

There are some early ideas to deprecate that hook generator libraries to push people to use DocumentNode’s.

Reply:
I will do the switch, thanks for the input!

Reply:
I'm happy to help if you're up against time.

Reply:
Yes, feel free!

Reply:
Done ✅